docs/concepts/work-pools.md

```diff
@@ -263,3 +301,71 @@ Together work queue priority and concurrency enable precise control over work. F
 
 ### Local debugging
 As long as your deployment's infrastructure block supports it, you can use work pools to temporarily send runs to an agent running on your local machine for debugging by running `prefect agent start -p my-local-machine` and updating the deployment's work pool to `my-local-machine`.
+
+## Worker Overview <span class="badge beta"></span>
+!!! warning "Workers are a beta feature"
+    Workers are a beta feature and are subject to change in future releases.
+
+Workers are lightweight polling services that retrieve scheduled runs from a work pool and execute them.
+
+Workers are similar to agents, but offer greater configuration options and the ability to route work to specific execution environments.
+
+Workers each have a type corresponding to the execution environment to which they will submit flow runs. Workers are only able to join work pools that match their type. As a result, when deployments are assigned to a work pool, you know in which execution environment scheduled flow runs for that deployment will be scheduled.
+
+### Worker Options
+Workers poll for work from one or more queues within a work pool. If the worker references a work queue that doesn't exist, it will be created automatically. The worker CLI is able to infer the worker type from the work pool, but you can also specify the worker type explicitly. If you supply the worker type to the worker CLI, a work pool will be created automatically if it doesn't exist.
```

Comment:
```suggestion
Workers poll for work from one or more queues within a work pool. If the worker references a work queue that doesn't exist, it will be created automatically. The worker CLI is able to infer the worker type from the work pool, but you can also specify the worker type explicitly. If you supply the worker type to the worker CLI, a work pool will be created automatically if it doesn't exist (using default job settings).
```